From 42bb2a7a998bcaffd780471ef0d442207653656f Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Tue, 2 Apr 2019 19:52:56 +0200
Subject: [PATCH 2/3] drbd_nl: commented out opener timestamp code.

We don't need it at the moment and would have to implement
the struct timeval and some timer function. We'll enable
that later when there is more time to do it.
---
 drbd/drbd_nl.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drbd/drbd_nl.c b/drbd/drbd_nl.c
index cdcd285..06c5a32 100644
--- a/drbd/drbd_nl.c
+++ b/drbd/drbd_nl.c
@@ -1307,6 +1307,7 @@ static void opener_info(struct drbd_resource *resource,
 
 	mutex_lock(&resource->open_release);
 
+#if 0
 	idr_for_each_entry(struct drbd_device *, &resource->devices, device, i) {
 		struct timespec64 ts;
 		struct tm tm;
@@ -1332,6 +1333,7 @@ static void opener_info(struct drbd_resource *resource,
 				      ts.tv_nsec / NSEC_PER_MSEC);
 		break;
 	}
+#endif
 	mutex_unlock(&resource->open_release);
 }
 
-- 
2.17.0

